### PR TITLE
Plugin promotions, deprecations and deletions

### PR DESCRIPTION
**Pull request by zwoop**
This does not move the slice plugin yet, since we have at least one serious fix to land before we move it.

### CODE REVIEW

File: doc/admin-guide/plugins/healthchecks.en.rst [hunk 1]
```diff
@@ -22,6 +22,9 @@
 Health Checks Plugin
 ********************
 
+**Note:** As of ATS v10.0.0, this pugin is deprecated. Please migrate over
```
Comment (shukitchan) [inline doc/admin-guide/plugins/healthchecks.en.rst:25]: typo - plugin?

File: doc/admin-guide/plugins/icap.en.rst [hunk 2]
```diff
@@ -22,6 +22,8 @@ ICAP Plugin
   under the License.
 
 
+**Note:** As of ATS v10.0.0, this pugin is deprecated and unsupported.
```
Comment (shukitchan) [inline doc/admin-guide/plugins/icap.en.rst:25]: typo - plugin?

**Comment by shukitchan** [comment]
@djcarlin , the healthcheck plugin will be gone in 10, replaced by statichit

File: doc/admin-guide/plugins/icap.en.rst (hunk 2 above)
Comment (zwoop) [reply doc/admin-guide/plugins/icap.en.rst:25]: Copy paste :)

**Comment by zwoop** [comment]
> @djcarlin , the healthcheck plugin will be gone in 10, replaced by statichit

Well, no, deprecated in ATS v10, removed in v11. We strongly recommend people moving over to statichit, and file issues and requests to add whatever features statichit may be missing.

**Comment by shukitchan** [comment]
> > @djcarlin , the healthcheck plugin will be gone in 10, replaced by statichit
> 
> Well, no, deprecated in ATS v10, removed in v11. We strongly recommend people moving over to statichit, and file issues and requests to add whatever features statichit may be missing.

My mistake. It is deprecated, not removed in 10.

File: doc/locale/ja/LC_MESSAGES/admin-guide/plugins/index.en.po [hunk 3]
```diff
@@ -338,10 +328,6 @@ msgid ""
 "using the X-Debug header."
 msgstr ""
 
-#: ../../../admin-guide/plugins/index.en.rst:202
-msgid ":doc:`Collapsed-Forwarding <collapsed_forwarding.en>`"
-msgstr ""
-
```
Comment (shukitchan) [inline doc/locale/ja/LC_MESSAGES/admin-guide/plugins/index.en.po:344]: next 5 lines need to be removed, too.

**Comment by shukitchan** [comment]
In doc/admin-guide/plugins/index.en.rst, we need to move the promoted plugins to the right section.

Everything else looks fine.

**Comment by shukitchan** [comment]
We should add a doc for statichit later on.

**Review by shukitchan — DISMISSED** [review]

**Comment by zwoop** [comment]
@shukitchan got merge conflicts again of course, so need a new review please :).

**Review by shukitchan — APPROVED** [review]

**Pull request merged**